distributor-widget/getting-started.md

```diff
@@ -71,8 +71,7 @@ Please double-check that you've added the script as instructed and followed all
 If you have a Content Security Policy (CSP) setup on your website, the following domains should be enabled for Distributor to function correctly.
 
 ```text
-mews.li
-*.mews.li
+*.mews.com
```

Comment:
If I understand correctly, with this we are making it more strict. Previously it would allow among others also `https://mews.li`, now we'd allow only subdomains. 

At least based on 2017-10-13 update in:
https://stackoverflow.com/a/44857848/11465224

and linked issue and spec: 
https://github.com/w3c/webappsec-csp/issues/241
https://w3c.github.io/webappsec-csp/#source-expression

Do we really want it more strict? 

Similar question for the link in `distributor-widget/integrations.md`

Reply:
This is not making anything more strict,

we do load content to the website from subdomains which can be:
app.mews.com
apps.mews.com
api.mews.com
cdn.mews.com
...

I don't see the point of including the `mews.com` domain, since the mews.com is only for our company presentation (marketing web) and no content should be loaded from that. And anyway if it would need to be it can be `www.mews.com`

Reply:
So it's more strict than before. But we don't want to allow it because it doesn't make sense for us. Works for me.